### PR TITLE
EOS-18850: Update py-utils code wrt Elasticsearch and Kibana OpenDistro v7.10

### PR DESCRIPTION
**Pull request by avrnv**
# Backend
## Problem Statement
Story Ref (if any): [EOS-18850](https://jts.seagate.com/browse/EOS-18850)
## Unit testing on RPM done
Yes
## Problem Description
Migration from ElasticSearch 6 to open distro ElasticSearch 7.10.
## Solution
Remove mapping types references from ElasticSearch client's internal implemenation.
## Unit Test Cases
- run py-utils/src/utils/data/db/examples/elasticsearch_storage.py

### CODE REVIEW

File: py-utils/requirements.txt [hunk 1]
```diff
@@ -11,5 +11,5 @@ numpy==1.19.2
 matplotlib==3.1.3
 confluent-kafka==1.5.0
 python-crontab==2.5.1
-elasticsearch==6.8.1
+elasticsearch>=7.0.0,<8.0.0
```
Comment (udayan-y21) [inline py-utils/requirements.txt:14]: I guess we also need to update.. Global sitepackages https://github.com/Seagate/cortx-re/blob/main/scripts/third-party-rpm/python-requirements.txt and CSM Local reference python packages..

Also `>=7.0.0,<8.0.0` will not match the version from pip3 freeze which will fail package validation for mini provisioning.

@kupranay @pawankumarsrivastava Your review on this.

**Review by udayan-y21 — CHANGES_REQUESTED** [review]
Please address the comment provided below. https://github.com/Seagate/cortx-utils/pull/258#discussion_r611630151

File: py-utils/requirements.txt (hunk 1 above)
Comment (avrnv) [reply py-utils/requirements.txt]: Updated to the exact version from `pip freeze` output.

**Comment by hessio** [comment]
@avrnv is this PR still necessary, if not can we close it?

**Comment by avrnv** [comment]
> @avrnv is this PR still necessary, if not can we close it?

The PR is no longer necessary, closing.

**Pull request closed without merging**